Resources/Prototypes/Entities/Clothing/Shoes/misc.yml

```
@@ -148,3 +148,23 @@
     price: 75
   - type: Tag
     tags: [ ]
+   
+- type: entity
+  parent: ClothingShoesBase
+  id: ClothingShoesJumpBoots
```

```suggestion
  id: ClothingShoesBootsJump
```
should be named like this to have the same naming convention as ClothingShoesBootsMag, ClothingShoesBootsSalvage, ClothingShoesBootsMoon and so on